### ISSUE TITLE
alpine:3.6 has CVE-2019-14697

### ISSUE THREAD

**Issue by armaanshah96** [dabealu/zookeeper-exporter#14]
Hello,

Our vulnerability scanning tool detected [CVE-2019-14697](https://nvd.nist.gov/vuln/detail/CVE-2019-14697), a vulnerability in `musl`, which is used in alpine:3.6

```
Vulnerabilities
---------------
Image                                ID                  CVE               Package     Version       Severity    Status                 CVSS
-----                                --                  ---               -------     -------       --------    ------                 ----
dabealu/zookeeper-exporter:latest    3366c4c4d651f5d4    CVE-2019-14697    musl        1.1.16-r15    critical                           9.8

```

**Comment by armaanshah96** [comment]
I created a PR to update the alpine version. Let me know if I missed something though in considering the update.

**Comment by dabealu** [comment]
Hi @armaanshah96 , thanks for your finding, I left a comment to PR.

**Comment by dabealu** [comment]
This fix included in image tag `v0.1.2` or newer.
